roles/kubernetes-apps/ingress_controller/ingress_nginx/README.md

```diff
@@ -50,7 +50,7 @@ First you need to [enable kubernetes](https://docs.docker.com/docker-for-mac/#ku
 Then you have to create a service:
 
 ```console
-kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/static/provider/cloud-generic.yaml
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/nginx-0.30.0/deploy/static/provider/cloud-generic.yaml
```

Comment:
The reason why the broken likes is the file cloud-generic.yaml got old and has been removed since https://github.com/kubernetes/ingress-nginx/pull/5399 on ingress-nginx repo.
So do we need to keep this?

Reply:
Not sure we do, agreed with you